app/web/template/layout/lpa-decisions.gohtml

```diff
@@ -70,25 +76,29 @@
                     {{ .Lpa.ReplacementAttorneysFullNames }}
                 {{ end }}
             </dd>
-            <dd class="govuk-summary-list__actions">
-                <a class="govuk-link" href="{{ link .App .App.Paths.ChooseReplacementAttorneys }}">
-                    {{ tr .App "change" }}<span class="govuk-visually-hidden">{{ tr .App "yourReplacementAttorneys" }}</span>
-                </a>
-            </dd>
+            {{ if not (eq .Lpa.Tasks.CheckYourLpa.String "completed") }}
+                <dd class="govuk-summary-list__actions">
+                    <a class="govuk-link" href="{{ link .App .App.Paths.ChooseReplacementAttorneys }}">
+                        {{ tr .App "change" }}<span class="govuk-visually-hidden">{{ tr .App "yourReplacementAttorneys" }}</span>
+                    </a>
+                </dd>
+            {{ end }}
         </div>
 
         <div class="govuk-summary-list__row">
             <dt class="govuk-summary-list__key">
                 {{ tr .App "restrictions" }}
             </dt>
             <dd class="govuk-summary-list__value">
-                <p style="white-space: pre-line;">{{ .Lpa.Restrictions }}</p>
-            </dd>
-            <dd class="govuk-summary-list__actions">
-                <a class="govuk-link" href="{{ link .App .App.Paths.Restrictions }}">
-                    {{ tr .App "change" }}<span class="govuk-visually-hidden">  {{ tr .App "yourRestrictions" }}</span>
-                </a>
+                <p class="govuk-body">{{ .Lpa.Restrictions }}</p>
             </dd>
+            {{ if not (eq .Lpa.Tasks.CheckYourLpa.String "completed") }}
+                <dd class="govuk-summary-list__actions">
+                    <a class="govuk-link" href="{{ link .App .App.Paths.Restrictions }}">
+                        {{ tr .App "change" }}<span class="govuk-visually-hidden">  {{ tr .App "yourRestrictions" }}</span>
```

Comment:
at some point we'll want these to return you to the previous page, it's going to be so annoying to do

Reply:
😿